application-diagram-ui/src/main/resources/Diagram/DiagramSheet.xml

```diff
@@ -634,15 +634,32 @@ define('diagram-graph-xml-filter', ['jquery', 'diagram-config', 'diagram-link-ha
   };
 
   // Replace custom internal links with absolute URLs (but keep some metadata to indicate the entity reference).
+  // Consider both the cases when the node is UserObject or mxCell.
   var onExportLink = function(node) {
-    var link = $(node).attr('link');
+    var tagName = node.tagName.toLowerCase();
+    var value = $(node).attr('value');
+    if (value) {
+      var valueInsideTag = value.match(/&lt;a\shref=".*"&gt;/g)[0];
+    }
```

Comment:
If the value is HTML then you can also use:

    $($(node).attr('value')).find('a[href]').attr('href')

I would avoid using regular expressions in this case.

Reply:
For the case when I would have text before the link like in _"Text `<a href="data:xwiki/reference..">link</a>` other text"_  I will have an error for `$($(node).attr('value'))`.
Is it a good choice to wrap `$(node).attr('value')` with a div and make the search inside this node?

Reply:
Yes, what I usually do is:

    $('<div/>').html('...').find(...)